recipes/openquake/meta.yaml

```diff
@@ -0,0 +1,290 @@
+{% set version = "2.8.0" %}
+
+package:
+  name: openquake.engine
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/o/openquake.engine/openquake.engine-{{ version }}.tar.gz
+  sha256: 55aed57bb65395e733106e5a1b2cb3fde9d8966f40f3b02167ccc9446d356473
+
+build:
+  number: 0
+  skip: True  # [not py35]
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
+
+requirements:
+  build:
+    - python >3.5.0,<3.5.9
+    - pip
+    - mock >=1.0,<1.4
+    - h5py >=2.2,<2.8
+    - nose >=1.3,<1.4
```

Comment:
Same question as above for `nose`.

Reply:
That came from the original setup script, not written by me:

https://github.com/gem/oq-engine/blob/master/setup.py#L56